akka-docs/rst/java/persistence.rst

```diff
@@ -575,7 +575,7 @@ To send messages with at-least-once delivery semantics to destinations you can e
 class instead of ``UntypedPersistentActor`` on the sending side.  It takes care of re-sending messages when they
 have not been confirmed within a configurable timeout.
 
-The state of the sending actor, including which messages that have been sent and still not been
+The state of the sending actor, including messages that have been sent and still not been
```

Comment:
Here you picked the wrong possible interpretation: it should be “including which messages have been sent” because it is the knowledge about their status, not the messages themselves, that is needed.